source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -1572,26 +1540,33 @@ void ShaderGraph::setVariableNames(GenContext& context)
     }
 }
 
-void ShaderGraph::populateInputColorTransformMap(ColorManagementSystemPtr colorManagementSystem, ShaderNodePtr shaderNode, ValueElementPtr input, const string& targetColorSpace)
+void ShaderGraph::populateColorTransformMap(bool asInput, ColorManagementSystemPtr colorManagementSystem, ShaderPort* shaderPort, 
```

Comment:
This makes it so one routine can handle both input and outputs (same as what was done for units)